cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   External Site Initialization
+   Used on at least `/external-site/`.
+   ========================================================================== */
+
+'use strict';
+
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * ExternalSite
+ * @class
+ *
+ * @classdesc Initializes a new ExternalSite instance.
+ *
+ * @param {HTMLElement} element DOM Element.
+ */
+function ExternalSite( element ) {
+
+  var BASE_CLASS = 'external-site_container';
+  var TOTAL_DURATION = 5;
+  var INTERVAL = 1000;
+
+  var _dom =
+    atomicCheckers.validateDomElement( element, BASE_CLASS, 'ExternalSite' );
+  var _durationEl = _dom.querySelector( '.external-site_reload-container' );
+  var _directEl = _dom.querySelector( '.external-site_proceed-btn' );
+  var _duration = TOTAL_DURATION;
+  var intervalId;
```

Comment:
Yup! Fixed in https://github.com/cfpb/cfgov-refresh/commit/6c959260085879dfbb0fccb51b9a0764db7d2f56